docs/release-notes.rst

```diff
@@ -72,6 +72,12 @@ For each release, we list the following types of change (in this order):
    in the "stable" docs (corresponding to the last release) but will be visible in the
    "latest" docs (corresponding to the master branch).
 
+Bug Fixes
+---------
+
+- Parallel execution introduced in 0.8.0 had a bug in logging, where logging breaks
+  when at least one of the arguments to the log message is unpickleable.
```

Comment:
We might want to expand on "logging breaks": this causes lost log messages and warnings, but not a crash, right?

Reply:
That's correct. I'll take another stab at this.